addons/website_livechat/models/discuss_channel.py

```diff
@@ -61,9 +61,9 @@ def _get_visitor_history(self, visitor):
     def _get_visitor_leave_message(self, operator=False, cancel=False):
         if cancel:
             name = self.livechat_visitor_id.display_name or _('The visitor')
```

Comment:
`The visitor`  should be part of the sentence when there is no specific value, so at least 2 translations should be provided here.

Reply:
I've fixed it for "The visitor", but there's still "an operator" which might be wrong in some languages, but hopefully it'll be fine. I won't touch it further unless someone reports a problem

Reply:
Good catch, I missed the other part. It's strange to do only half of it, why is "the visitor" worth the change but not "an operator"?

Reply:
You're right, let's do nothing then